__init__.py

```diff
@@ -25,6 +25,61 @@
 from mycroft.util.parse import normalize
 from mycroft import intent_file_handler
 
+from PIL import Image, ImageDraw, ImageFont
+import struct
+
+# Basic drawing to the framebuffer
+BACKGROUND = (34, 167, 240)
+FONT_PATH = 'NotoSansDisplay-Bold.ttf'
+
+
+def fit_font(text, font_path, font_size):
+    """ Brute force a good fontsize to make text fit screen. """
+    font = ImageFont.truetype(font_path, font_size)
+    w, h = font.getsize(text)
+    while w < 0.9 * 400:
+        # iterate until the text size is just larger than the criteria
+        font_size += 1
+        font = ImageFont.truetype(font_path, font_size)
+        w, h = font.getsize(text)
+
+    return font
+
+
+def write_fb(im, dev=None):
+    """ Write Image Object to framebuffer.
+
+        TODO: Check memory mapping
+    """
+    dev = dev or '/dev/fb0'
```

Comment:
How about `dev='/dev/fb0'` as default instead of `None` ?

Reply:
Will do, I'm never sure which type is mutable and which is immutable so I generally set the default to None to be sure :)